modules/rsem/preparereference/main.nf

```diff
@@ -40,6 +40,8 @@ process RSEM_PREPAREREFERENCE {
             $fasta \\
             rsem/genome
 
+        cp rsem/genome.transcripts.fa .
```

Comment:
Doesn't this collide with the input?

Reply:
Never mind. I apparently cannot tell the difference between the words `input` and `output`.